docs/topics/maven.md

```diff
@@ -27,11 +27,10 @@ To use the standard library in your project, add the following dependency in the
 </dependencies>
 ```
 
-If you're targeting JDK 7 or JDK 8, you can use extended versions of the Kotlin standard library. They contain
-additional extension functions for APIs added in new JDK versions. Instead of `kotlin-stdlib`, use `kotlin-stdlib-jdk7`
-or `kotlin-stdlib-jdk8`, depending on your JDK version. 
-
->For Kotlin versions older than 1.2, use `kotlin-stdlib-jre7` and `kotlin-stdlib-jre8`.
+> For Kotlin versions older than:
+> * 1.8, if you're targeting JDK 7 or 8, use `kotlin-stdlib-jdk7`
+     or `kotlin-stdlib-jdk8`.
```

Comment:
Maybe to unite this line with the previous and add "respectively"?

`> * 1.8, if you're targeting JDK 7 or 8, use `kotlin-stdlib-jdk7` or `kotlin-stdlib-jdk8` respectively.`

Reply:
I incorporated your suggestion but also decided to change the format. Would you mind checking it again?